script/schema_doc.py

```
@@ -733,6 +736,36 @@ def oddities_titles(folder, file, title):
+                if filter_schema:
+                    see_also.reset_doc(md_file)
+                    while True:
+                        index, title = md_get_next_title(md_file, lines, index)
+                        if not title:
+                            break
+                        if title == DOC_CONFIGURATION_VARIABLES:
+                            try:
+                                index = process_config(
+                                    md_file, lines, index + 1, filter_schema
+                                )
+                            except Exception as err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)